__tests__/app.test.js

```diff
@@ -36,4 +44,28 @@ describe('# GET REQUESTS', () => {
                 });
         });
     });
+
+    describe('GET /api/atricles/:article_id', () => {
+
+        test('200: Returns list of articles', () => {
```

Comment:
This endpoint is requesting a single article. It should return that article object on a key of `article`. The test you have here is more appropriate for an endpoint like `GET a/pi/articles`. 

Refactor this to match the expected response from the ticket.